controllers/release/adapter.go

```diff
@@ -302,13 +302,13 @@ func (a *Adapter) EnsureReleaseProcessingIsTracked() (reconciler.OperationResult
 // PipelineRun.
 func (a *Adapter) createReleasePipelineRun(resources *loader.ProcessingResources) (*v1beta1.PipelineRun, error) {
 	pipelineRun := tekton.NewReleasePipelineRun("release-pipelinerun", resources.ReleaseStrategy.Namespace).
-		WithObjectReferences(a.release, resources.ReleasePlanAdmission).
+		WithObjectReferences(a.release, resources.ReleasePlan,
+			resources.ReleasePlanAdmission, resources.ReleaseStrategy, resources.Snapshot).
```

Comment:
This also adds the `ReleaseStrategy`, right? Can you change the commit message / PR description to that effect?

Reply:
done

Reply:
Thanks.